src/components/Result.tsx

```diff
@@ -37,11 +43,30 @@ const Result = () => {
     };
 
     return (
-        <div>
-            <p>Question : {result.question.questionNumber}</p>
-            <p>Your answer : {answerId}</p>
-            <p>Answer 1: %{result.firstAnswerRate}</p>
-            <p>Answer2: %{result.secondAnswerRate}</p>
+        <div className='result__container'>
+            <div className="result__inner-container">
```

Comment:
So then here it would be result__inner-container or result-container__inner (parent name + __ + child name)